walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,54 +14,47 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Comparator;
 import java.util.List;
-import java.util.ArrayList;
 import java.util.Set;
 
 public final class FindMeetingQuery {
 
-    public static final int START_OF_DAY = TimeRange.START_OF_DAY;
-    public static final int END_OF_DAY = TimeRange.END_OF_DAY;
-    public static final TimeRange WHOLE_DAY = TimeRange.WHOLE_DAY;
-    public static final Comparator<TimeRange> ORDER_BY_START = TimeRange.ORDER_BY_START;
-
+    private static final int START_OF_DAY = TimeRange.START_OF_DAY;
+    private static final int END_OF_DAY = TimeRange.END_OF_DAY;
+    private static final TimeRange WHOLE_DAY = TimeRange.WHOLE_DAY;
+    private static final Comparator<TimeRange> ORDER_BY_START = TimeRange.ORDER_BY_START;
 
     public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
 
         // Array of queries represenitng times to meet.
         List<TimeRange> query = new ArrayList<>();
         
-        // Integer represenitng the number of events.
         int numberOfEvents = events.size();
         
-        // Integer representing the Start of a work day.
         int timeOfStart = START_OF_DAY;
-
-        // Integer representing the End of a work day.
-        int timeOfEnd = END_OF_DAY;
         
-        // Integer represenig the duration of a meeting request.
         int desiredDuration = (int) request.getDuration();
         
         //  Event counter used whilst iterating.
         int eventCounter = 0;
         
-        int event_start = 0;
-        int event_end = 0;
+        int eventStart = 0;
+        int eventEnd = 0;
 
         // Set of stirngs representing unique employees set to attend this meeting.
-        Set<String> event_attendees = null;
+        Set<String> attendees = new Set();
```

Comment:
Set is an abstract class and can't be directly instantiated. You'll want one of its concrete subclasses like HashSet.

Reply:
Ack